uber/templates/base.html

```diff
@@ -113,15 +113,6 @@
                     errorbox_y: 28
                 });
 
-                $('.jq-dte').each(function(index, element) {
```

Comment:
That's something I added in an earlier pull request. The jQuery datetextentry plugin rewrites the HTML to make the original input field a hidden input field. Then it adds three new inputs for month, day, year. This code removes the `required` property from the original input field and applies it to the three newly generated inputs.

But we don't want any of them to have the `required` property, so this code is no longer useful.